apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.Options.cs

```diff
@@ -174,15 +195,31 @@ public sealed class Options
             /// <param name="urlStyle">The new url style.</param>
             /// <returns>A new set ofoptions with the given url style.</returns>
             public Options WithUrlStyle(UrlStyle urlStyle) =>
-                new Options(Duration, Expiration, SigningVersion, urlStyle, Scheme, null);
+                new Options(Duration, Expiration, SigningVersion, urlStyle, Scheme, Host, Port, null);
```

Comment:
Possibly note that this clears BucketBoundHostname? I'm pretty sure that's deliberate, but it confused me for a while.